### PR TITLE
fix(regex): fix data file for 942280 rule

### PR DESCRIPTION
**Pull request by theMiddleBlue**
(sorry for the fork branch name) refer to #2929

### CODE REVIEW

**Comment by theMiddleBlue** [comment]
I think it's ready to be merged

File: util/regexp-assemble/data/942280.data [hunk 1]
```diff
@@ -3,10 +3,18 @@
...
 
+##!> template semicolon ;
+
+##! The following part is going to be appended
+##! to match the end of a query or a comment syntax
+##!> template sql_comments (?:[#{{semicolon}}{]|\/\*|--)
+
+##! In PostgreSQL, you can use the pg_sleep() function
+##! to delay execution for a given number of seconds
 select\s*?pg_sleep
```
Comment (fzipi) [inline util/regexp-assemble/data/942280.data:14]: Maybe you want to add a template like `optional-empty-space \s*?` ? If not, LGTM.

**Review by fzipi — DISMISSED** [review]

File: util/regexp-assemble/data/942280.data (hunk 1 above)
Comment (theMiddleBlue) [reply util/regexp-assemble/data/942280.data:14]: that's a good idea, going to add it

File: util/regexp-assemble/data/942280.data [hunk 2]
```diff
@@ -3,10 +3,18 @@
 
 ##!+ i
 
+##!> template semicolon ;
```
Comment (theseion) [inline util/regexp-assemble/data/942280.data:6]: It's obvious that it's a semicolon. Either change the name or add a comment to indicate the semantics, e.g., `single-line-semicolon-comment`.
Is there something special about semicolon? Why isn't `--` an option here?

**Review by theseion — CHANGES_REQUESTED** [review]

File: util/regexp-assemble/data/942280.data (hunk 2 above)
Comment (theMiddleBlue) [reply util/regexp-assemble/data/942280.data:6]: maybe we can keep the semicolon as is without a template? is it better?

File: util/regexp-assemble/data/942280.data [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
 ##! The following part is going to be appended
 ##! to match the end of a query or a comment syntax
-##!> template sql_comments (?:[#;{]|\/\*|--)
+##!> template terminates (?:[#;{]|\/\*|--)
```
Comment (theseion) [inline util/regexp-assemble/data/942280.data:10]:
```suggestion
##!> template terminators (?:[#;{]|\/\*|--)
```

**Review by theseion — CHANGES_REQUESTED** [review]

**Comment by fzipi** [comment]
@theMiddleBlue Looks like we have conflicts now.

File: util/regexp-assemble/data/942280.data [hunk 4]
```diff
@@ -3,10 +3,18 @@
...
 ##!+ i
 
-select\s*?pg_sleep
-waitfor\s*?delay\s?[\"'`]+\s?\d
-;\s*?shutdown\s*?;
-;\s*?shutdown\s*?--
-;\s*?shutdown\s*?#
-;\s*?shutdown\s*?\/\*
-;\s*?shutdown\s*?{
+##!> template optional_whitespace \s*?
```
Comment (theseion) [inline util/regexp-assemble/data/942280.data:6]:
```suggestion
##!> define optional_whitespace \s*?
```

**Review by theseion — DISMISSED** [review]
You'll need to rebase. The file is now in a different directory and has the new `.ra` extension.

**Comment by theMiddleBlue** [comment]
it should be ok now

**Review by fzipi — APPROVED** [review]
LGTM

**Pull request merged**